tests/unit/how-to-run-tests.md

```diff
@@ -0,0 +1,11 @@
+How to run unit tests
```

Comment:
I would prefer we put testing instructions in the `README.md` as it should be the central point for all big info in our project